src/uu/ln/src/ln.rs

```diff
@@ -388,7 +388,11 @@ fn relative_path<'a>(src: &Path, dst: &Path) -> Result<Cow<'a, Path>> {
         .map(|_| OsStr::new(".."))
         .chain(src_iter)
         .collect();
-    Ok(result.into())
+    if result.as_os_str().is_empty() {
+        Ok(PathBuf::from(".").into())
```

Comment:
I guess I could append the `.` to the original `result` instead of creating a new `PathBuf`. I'll change that tomorrow.